cadc-vosi/src/main/java/ca/nrc/cadc/vosi/avail/CheckCertificate.java

```diff
@@ -112,38 +114,42 @@ public CheckCertificate(File cert) {
     public CheckCertificate(File cert, File key) {
         this.cert = cert;
         this.key = key;
+        this.certFilename = this.cert.getName();
```

Comment:
Don't really want to use redundant member vars like this... it's better to just use `cert..getAbsolutePath()` when you want to add it to a message, because that message content will also be immediately clear.